acceptance/telemetry/upload-fails/script

```diff
@@ -0,0 +1,12 @@
+export DATABRICKS_CLI_TELEMETRY_PID_FILE=./telemetry.pid
+export DATABRICKS_CLI_TELEMETRY_UPLOAD_LOGS_FILE=./out.upload_process.txt
+
+# This test ensures that the main CLI command does not error even if
+# telemetry upload fails.
+trace $CLI selftest send-telemetry
+
+# Wait for the child telemetry process to finish
+wait_pid.py $(cat ./telemetry.pid)
```

Comment:
nit: I'd consider wait_pid.py accepting filename directly and read it from python; better error messages in that case.